### PR TITLE
LPS-157822 Test Automation Creation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-157822

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/SelectFromList.testcase [hunk 1]
```diff
@@ -249,6 +251,250 @@ definition {
...
+				CreateObject.selectTitleField(fieldLabel = "Lunch");
+
+				CreateObject.saveObject();
+
+				ObjectAdmin.gotoRelationshipsTab();
+
+				ObjectAdmin.addObjectRelationshipViaUI(
+					label = "Lunch Menu",
+					object = "PurchaseOrder",
+					type = "One to Many");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/SelectFromList.testcase:300]: Use ObjectAdmin.addObjectRelationshipViaAPI

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/SelectFromList.testcase [hunk 2]
```diff
@@ -249,6 +251,250 @@ definition {
...
+			}
+
+			task ("Add a Form Container framgment to content page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/SelectFromList.testcase:303]: task ("Add a Form Container fragment to content page") {

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/SelectFromList.testcase [hunk 3]
```diff
@@ -249,6 +251,250 @@ definition {
...
+				CreateObject.selectTitleField(fieldLabel = "Name");
+
+				CreateObject.saveObject();
+
+				ObjectAdmin.gotoRelationshipsTab();
+
+				ObjectAdmin.addObjectRelationshipViaUI(
+					label = "Account Name",
+					object = "PurchaseOrder",
+					type = "One to Many");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/SelectFromList.testcase:402]: Same as above

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Dropdown.path [hunk 4]
```diff
@@ -55,6 +55,11 @@
 	<td>//div[contains(@class,'dropdown-menu')][contains(@class,'show')]//div[contains(@class,'alert-info')][normalize-space(text())='${infoMessage}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>INPUT_SEARCH_BAR</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Dropdown.path:59]: Use MenuItem#SEARCH_FIELD
`//div[contains(@class,'dropdown-menu show')]//input[contains(@type,'text')]`

There should be only one dropdown shows on page

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments?

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Dropdown.path (hunk 4 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Dropdown.path:59]: Thanks! I didn't find this one :(

Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Dropdown.path:59]: About this path it doesn't work for this search bar, the type is _search_, so I just added a variable on the path and updated the usages

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments?

Hi @Tim-Cao! PR updated, would you have a look at? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/MenuItem.path [hunk 5]
```diff
@@ -67,7 +67,7 @@
 </tr>
 <tr>
 	<td>SEARCH_FIELD</td>
-	<td>//div[contains(@class,'dropdown-menu show')]//input[contains(@type,'text')]</td>
+	<td>//div[contains(@class,'dropdown-menu show')]//input[contains(@type,'${key_type}')]</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/MenuItem.path:70]: Use
`//div[contains(@class,'dropdown-menu show')]//input[.//following-sibling::span//*[name()='svg'][contains(@class,'icon-search')]]`

**Comment by Tim-Cao** [comment]
See changes @ozysouza
https://github.com/liferay-echo/liferay-portal/pull/8936

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**